### ISSUE TITLE
Accesible forms

### ISSUE THREAD

**Issue by Glombort** [fac24/week2-alex-maria-peter-sonia#26]
I think A11y suggests to use ARIA labels on forms for even more help for accessible uses of the form elements

**Comment by oliverjam** [comment]
ARIA should generally be used as a last resort if the page cannot be made accessible without it. In this case all the form elements have visible labels, so adding extra `aria-label`s would at best be redundant and at worst confusing for screen reader users.

E.g.
```html
<label for="username">Username:</label>
<input id="username" name="username" aria-label="Enter your username" />
```
would be a bit weird for a user who could see _and_ uses a screen reader (most legally blind people still have some vision). They would _see_ "Username" but hear something different.

Assuming your visible labels are correctly connected to the input (with `for="id"`) then assistive tech should figure out the label fine.
